CHANGELOG.markdown

```diff
@@ -4,6 +4,8 @@
 * Add `Control.Lens.Profunctor` with conversion functions to and from
   profunctor optic representation
 * Mark `Control.Lens.Equality` as Trustworthy
+* `Swapped` type-class is removed in favor of `Swap` from `assoc` package
```

Comment:
These changelog entries should briefly describe how to migrate from `lens:Swapped` and `lens:Strict` to `assoc:Swap` and `strict:Strict`, respectively.